lib/utils/reify-output.js

```diff
@@ -33,11 +33,15 @@ const reifyOutput = (npm, arb) => {
   }
 
   const summary = {
+    add: [],
     added: 0,
-    removed: 0,
-    changed: 0,
+    // audit gets added later
     audited: auditReport && !auditReport.error ? actualTree.inventory.size : 0,
+    change: [],
+    changed: 0,
```

Comment:
? why are we keeping a separate `count` of these guys instead of just counting the # of items in the arrays when we need to?

Reply:
removing `changed` would be a breaking change.  This was the only way to do it w/o waiting for npm 12.

The reason this looks like two additions is cause I sorted them.